lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,82 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        include AbstractType
+
+        REGISTRY = {} # rubocop:disable MutableConstant
```

Comment:
> This was the interface before which is why I preserved it

Ahh I see. @mbj any reason for the separation? I could see there being a reason iff the `Registry` was going to be refactored to have different kinds of instances injected into it.

Assuming there's not a compelling reason I would recommend collapsing the implementation and exposing only the `Registry` public methods in the interface.